android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageFacebookConceal.java

```diff
@@ -112,30 +113,25 @@ public DecryptionResult decrypt(@NonNull final String alias,
       final byte[] decryptedUsername = crypto.decrypt(username, usernameEntity);
       final byte[] decryptedPassword = crypto.decrypt(password, passwordEntity);
 
-      return new DecryptionResult(
+      return CompletableFuture.supplyAsync(() -> new DecryptionResult(
```

Comment:
What do you think to include `CompletableFuture` compat version? Like what is suggested here: https://stackoverflow.com/a/38375991 , or maybe I can try to write my own wrapper (because it seems that multithreading that CompletableFuture provide is not that important for the task)

Reply:
This compat version seems to be not official one. We will have maintenance issues later on.

Reply:
@savelichalex We tested this patch and observed it fixes the crash issue, writing own wrapper sounds good to me will you be able to update this pr.

Reply:
@ankitsingh08 when I tried to make the wrapper turned out it's not a 20 minutes ride :D I have an idea to make it with callbacks, unfortunately I don't have much time for this right now, but I'll try to allocate some time for it